assets/js/components/link.js

```diff
@@ -73,6 +74,12 @@ class Link extends Component {
 				{ ...extraProps }
 			>
 				{ children }
+
+				{ external && (
+					<span className="screen-reader-text">
+						{ _x( '(opens in a new tab)', 'screen reader text', 'google-site-kit' ) }
```

Comment:
Tested this in a screenreader and it didn't quite work.

We need either:

```suggestion
						{ _x( ' (opens in a new tab)', 'screen reader text', 'google-site-kit' ) }
```

or 

```suggestion
						{ ' ' }{ _x( '(opens in a new tab)', 'screen reader text', 'google-site-kit' ) }
```

This is because otherwise React will remove the whitespace and text like:

```js
<Link className="googlesitekit-help-link" href={ url } external>Need help?</Link>
```

will output to: `Need help(opens in a new tab)`.

So there needs to be whitespace here to separate the text from the parens. I get caught by React's clever whitespace removal sometimes 😅 

I'd prefer using the separate whitespace, as it makes it easier for translators to understand and this approach already has some i18n downfalls, but is good enough for now.

After that this is good to go.